### ISSUE TITLE
Support the simple_pattern and simple_pattern_split tokenizers

### ISSUE THREAD

**Issue by tkirill** [elastic/elasticsearch-net#5716]
Hello!

Recently I wanted to use [simple_pattern_split](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-simplepatternsplit-tokenizer.html) tokenizer and didn't found a way to set it up with `TokenizerDescriptor`. I used `pattern` tokenizer instead but faster `simple_pattern_split` is more optimal for my case because the pattern is very simple.

So I thought maybe it will be a good idea to support `simple_pattern_split` and `simple_pattern` in Nest. Something like:

```
await elasticClient.Indices.CreateAsync(
    config.IndexName,
    c => c
        .Settings(s => s
            .Analysis(a => a
                .Tokenizers(t => t
                    .SimplePatternSplit("email", f => f.Pattern(@"\@|\."))))))
```

**Comment by stevejgordon** [comment]
Thanks for raising this @tkirill. This is certainly something we can and should look at supporting via NEST. I'll take a look at this.

**Comment by stevejgordon** [comment]
@tkirill I've merged in the inclusion of these tokenizers in NEST. Those will be available in the 7.14.0 release.
